test/controllers/api/auth/feeds_controller_test.rb

```diff
@@ -4,8 +4,8 @@
   let(:account_id) { 123 }
   let(:podcast) { create(:podcast, prx_account_uri: "/api/v1/accounts/#{account_id}") }
   let(:feed) { create(:feed, podcast: podcast, slug: "test-slug") }
-  let(:token) { StubToken.new(account_id, ["member"]) }
-  let(:bad_token) { StubToken.new(account_id + 100, ["member"]) }
+  let(:token) { StubToken.new(account_id, ["podcast_edit"]) }
```

Comment:
in this and the `FeedPolicyTest`, the scope now has to match how the Podcast scope is determined. in this case, the `FeedPolicy` actions are determined by the `update?` method, which uses the `PodcastPolicy.update?` method, which checks for the `:podcast_edit` scope, not just a `:member` scope.

Reply:
Ah, good catch!